server/cmd/execute.go

```diff
@@ -33,7 +32,7 @@ func Execute(rootCmd *cobra.Command, defaultHome string) error {
 	ctx = context.WithValue(ctx, client.ClientContextKey, &client.Context{})
 	ctx = context.WithValue(ctx, server.ServerContextKey, srvCtx)
 
-	rootCmd.PersistentFlags().String(flags.FlagLogLevel, zerolog.InfoLevel.String(), "The logging level (trace|debug|info|warn|error|fatal|panic)")
+	rootCmd.PersistentFlags().String(flags.FlagLogLevel, ostcfg.DefaultPackageLogLevels(), "The logging level by modules (default. main:info,state:info,statesync:info,*:info)")
```

Comment:
`ostcfg.DefaultPackageLogLevels()` returns `main:info,state:info,statesync:info,*:info`, and there is same description in the format. 

So `main:info,state:info,statesync:info,*:info` is displayed twice.

Reply:
How about rollback the description like before, not using default?